From 08c5150d76971ab17eb02ceef0e65e993b7732ff Mon Sep 17 00:00:00 2001
From: Stanislav Laznicka <slaznick@redhat.com>
Date: Fri, 21 Apr 2017 09:39:56 +0200
Subject: [PATCH] compat: ignore cn=topology,cn=ipa,cn=etc subtree

The entries in cn=topology,cn=ipa,cn=etc should not be taken in
account for the compat plugin.

https://pagure.io/freeipa/issue/6821

Reviewed-By: Martin Basti <mbasti@redhat.com>
---
 install/updates/80-schema_compat.update | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/install/updates/80-schema_compat.update b/install/updates/80-schema_compat.update
index 06cbcab8ad809d95a907c161044ff91df827ebf3..7483518134aad47195ab136626acb5130c0d536d 100644
--- a/install/updates/80-schema_compat.update
+++ b/install/updates/80-schema_compat.update
@@ -152,6 +152,7 @@ remove: schema-compat-ignore-subtree: o=ipaca
 add: schema-compat-restrict-subtree: $SUFFIX
 add: schema-compat-restrict-subtree: cn=Schema Compatibility,cn=plugins,cn=config
 add: schema-compat-ignore-subtree: cn=dna,cn=ipa,cn=etc,$SUFFIX
+add: schema-compat-ignore-subtree: cn=topology,cn=ipa,cn=etc,$SUFFIX
 
 # Change padding for host and userCategory so the pad returns the same value
 # as the original, '' or -.
@@ -162,6 +163,7 @@ remove: schema-compat-ignore-subtree: o=ipaca
 add: schema-compat-restrict-subtree: $SUFFIX
 add: schema-compat-restrict-subtree: cn=Schema Compatibility,cn=plugins,cn=config
 add: schema-compat-ignore-subtree: cn=dna,cn=ipa,cn=etc,$SUFFIX
+add: schema-compat-ignore-subtree: cn=topology,cn=ipa,cn=etc,$SUFFIX
 
 dn: cn=computers, cn=Schema Compatibility, cn=plugins, cn=config
 default:objectClass: top
@@ -181,6 +183,7 @@ remove: schema-compat-ignore-subtree: o=ipaca
 add: schema-compat-restrict-subtree: $SUFFIX
 add: schema-compat-restrict-subtree: cn=Schema Compatibility,cn=plugins,cn=config
 add: schema-compat-ignore-subtree: cn=dna,cn=ipa,cn=etc,$SUFFIX
+add: schema-compat-ignore-subtree: cn=topology,cn=ipa,cn=etc,$SUFFIX
 
 dn: cn=sudoers,cn=Schema Compatibility,cn=plugins,cn=config
 add:schema-compat-entry-attribute: sudoOrder=%{sudoOrder}
@@ -191,6 +194,7 @@ remove: schema-compat-ignore-subtree: o=ipaca
 add: schema-compat-restrict-subtree: $SUFFIX
 add: schema-compat-restrict-subtree: cn=Schema Compatibility,cn=plugins,cn=config
 add: schema-compat-ignore-subtree: cn=dna,cn=ipa,cn=etc,$SUFFIX
+add: schema-compat-ignore-subtree: cn=topology,cn=ipa,cn=etc,$SUFFIX
 
 dn: cn=groups,cn=Schema Compatibility,cn=plugins,cn=config
 remove: schema-compat-ignore-subtree: cn=changelog
@@ -198,6 +202,7 @@ remove: schema-compat-ignore-subtree: o=ipaca
 add: schema-compat-restrict-subtree: $SUFFIX
 add: schema-compat-restrict-subtree: cn=Schema Compatibility,cn=plugins,cn=config
 add: schema-compat-ignore-subtree: cn=dna,cn=ipa,cn=etc,$SUFFIX
+add: schema-compat-ignore-subtree: cn=topology,cn=ipa,cn=etc,$SUFFIX
 
 dn: cn=Schema Compatibility,cn=plugins,cn=config
 # We need to run schema-compat pre-bind callback before
-- 
2.12.2

